apps/frontend/src/components/questionary/questionaryComponents/ProposalBasis/QuestionaryComponentProposalBasis.tsx

```diff
@@ -116,8 +116,16 @@ function QuestionaryComponentProposalBasis(props: BasicComponentProps) {
           name={`${id}.abstract`}
           label="Proposal Abstract"
           inputProps={{
-            onChange: (event: ChangeEvent<HTMLInputElement>) =>
-              setLocalAbstract(event.target.value),
+            onChange: (event: ChangeEvent<HTMLInputElement>) => {

```

Comment:
question: Do you think using onPaste would be more efficient here? Assuming that user can not enter these invalid characters by keyboard
```
            onPaste: (e: React.ClipboardEvent<HTMLInputElement>) => {
              e.preventDefault();
              const raw = e.clipboardData.getData('text');
              const cleaned = raw.replace(/[^a-z0-9 ]/gi, '');
              setLocalTitle(cleaned);
            },
```